definitions/survey_responses.sqlx

```diff
@@ -12,14 +12,14 @@ SELECT
   src.survey_response_id,
   src.survey_wave_id,
   src.src_response_id,
-  src.date,
   src.weight,
   src.imd_quartile_country,
   lookup_gender.gender,
   lookup_age_group.age_group,
   lookup_qualification.qualification,
   lookup_ethnicity.ethnicity,
-  lookup_government_office_region.government_office_region
+  lookup_government_office_region.government_office_region,
```

Comment:
Do you think we start [publishing](https://github.com/alphagov/govuk-polling-dataform/pull/8/files#diff-683c9f8b39dc0450dc74a1365713a899ff0a880e2660db11811fb84e82843c9eR4) this to another dataset @oliverroberts1-gds? Would dataset `govuk-polling.govuk_polling_responses` be appropriate as it presumably already has the approriate access control set up?

Reply:
updated https://github.com/alphagov/govuk-polling-dataform/pull/8/commits/c8e9adc13eea1dc002dd5b52361c3a7ac44618fb